### ISSUE TITLE
List assignment

### ISSUE THREAD

**Issue by vinklein** [defeo/cypari2#44]
The PARI list objects are mutable. Though in cypari2, list assignment is silently doing nothing
```
>>> from cypari2 import Pari
>>> pari = Pari()
>>> l = pari.List([1,2,3])
>>> l[0] = 3
>>> l
List([1, 2, 3])
```
The equivalent session in GP
```
? L = List([1,2,3]);
? L[1] = 3;
? L
%3 = List([3, 2, 3])
```

**Comment by jdemeyer** [comment]
I guess nobody really bothered with supporting the type `t_LIST`, so it is not surprising that it doesn't work properly.

**Comment by videlec** [comment]
Fixed in #64
